allennlp/modules/token_embedders/embedding.py

```diff
@@ -194,7 +194,7 @@ def extend_vocab(
         model_path : `str`, (optional, default=None)
             Path traversing the model attributes upto this embedding module.
             Eg. "_text_field_embedder.token_embedder_tokens". This is only useful
-            to give helpful error message when extend_vocab is implicitly called
+            to give a helpful error message when extend_vocab is implicitly called
             by fine-tune or any other command.
```

Comment:
I noticed that `Embedding.extend_vocab` is used only in tests now. Should it be removed?

Reply:
It gets called here: https://github.com/allenai/allennlp/blob/c4559f3751775aa8bc018db417edc119d29d8051/allennlp/models/model.py#L364-L366.